python-lib/dku_plugin_config_loading.py

```diff
@@ -68,13 +67,29 @@ def _add_text_column(self):
         )
 
     def _add_matching_settings(self):
-        """Load matching parameters"""
+        """Load parameters in MATCHING PARAMETERS"""
 
         for parameter in self.MATCHING_PARAMETERS:
             self.dku_config.add_param(
                 name=parameter, value=self.config[parameter], required=True
             )
 
+    def _add_lemmatization(self, multilingual=False):
+        """Load one of the two lemmatization buttons:
+        -'lemmatization_multilingual' if language==language_column,
+        -'lemmatization' otherwise.
```

Comment:
If I understand correctly, this monolingual lemmatization is in fact english only, right ?

Reply:
The monolingual lemmatization is for all the 28 supported languages for lemmatization. We have made two lemmatization button because we wanted to have a different response from the UI, regarding on the case we have.

- -If the recipe is used with monolingual documents, the lemmatization button will be display only if the selected language is supported for lemmatization
- -If the recipe is used with documents of multilple languages, the lemmatization_multilingual  button will be display with an extra message (e.g "Available for 28 out of 59 languages."), and an error will be raised if one of the languages is not supported